examples/scripts/sft_trainer.py

```diff
@@ -99,7 +99,8 @@ class ScriptArguments:
     model=model,
     args=training_args,
     max_seq_length=script_args.seq_length,
-    train_dataset=dataset,
+    train_dataset=train_dataset,
+    eval_dataset=eval_dataset,
```

Comment:
this one should also be deleted. Please proofread your script once again as you apply changes.